src/main/java/edu/kis/powp/jobs2d/command/gui/backlog/BackLogItem.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.command.gui.backlog;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+public class BackLogItem {
+	private String date;
+	private String commandName;
+	private DriverCommand command;
+
+	public BackLogItem(String date, String commandName, final DriverCommand command) {
```

Comment:
Why only 'command' is final? 
Did you made sure, that the command injected cannot be modified from outside? (it look like you pass its reference)